src/main/kotlin/anttracker/db/SetupSchema.kt

```diff
@@ -134,7 +134,7 @@ class Release(
 }
 
 @JvmInline
-value class IssueDescription private constructor(
```

Comment:
The constructor is private to enforce usage of `maybeParse` so you are assured a valid description is always created.

Reply:
done

Reply:
or sorry, i mean reverted this change